### PR TITLE
Add support for `actions.disable_zaraz` in page rules

### LINKED ISSUE #1521: Add disable_zaraz param in cloudflare_page_rule resource
### Current Terraform and Cloudflare provider version

provider registry.terraform.io/cloudflare/cloudflare v3.10.1

### Description

Page rules allow disabling Zaraz but terraform resource don't have parameter to do it.

### Use cases

We manage all our page rules through terraform.

### Potential Terraform configuration

```hcl
resource "cloudflare_page_rule" "disable_zaraz" {
    zone_id  = XXX
    target   = XXX
    actions {
        disable_zaraz = true
    }
}
```

### References

_No response_

### PR DESCRIPTION
**Pull request by LANGERGabrielle**
Fixes #1521

### CODE REVIEW

File: cloudflare/resource_cloudflare_page_rule_test.go [hunk 1]
```diff
@@ -958,6 +988,17 @@ resource "cloudflare_page_rule" "test" {
...
+	return fmt.Sprintf(`
+resource "cloudflare_page_rule" "test" {
+	zone_id = "%s"
+	target = "%s"
+	actions {
+		disable_zaraz = true
+	}
+}`, zoneID, target)
+}
+
```
Comment (LANGERGabrielle) [inline cloudflare/resource_cloudflare_page_rule_test.go:1001]: I ran it and checked on the dashboard and it worked as expected :
image

Comment (LANGERGabrielle) [inline cloudflare/resource_cloudflare_page_rule_test.go:1001]: I ran it and checked on the dashboard and it worked as expected :
image

File: .changelog/1523.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+resource/resource/cloudflare_page_rule: add support for `actions.disable_zaraz`
```
Comment (jacobbednarz) [inline .changelog/1523.txt:2]:
```suggestion
resource/cloudflare_page_rule: add support for `actions.disable_zaraz`
```

**Comment by jacobbednarz** [comment]
thank you, this is a solid PR! acceptance tests are passing

```
TF_ACC=1 go test $(go list ./...) -v -run "^TestAccCloudflarePageRule_" -count 1 -parallel 1 -timeout 120m -parallel 1
?   	github.com/cloudflare/terraform-provider-cloudflare	[no test files]
=== RUN   TestAccCloudflarePageRule_Import
=== PAUSE TestAccCloudflarePageRule_Import
=== RUN   TestAccCloudflarePageRule_ImportWithBrowserCacheTTL30
--- PASS: TestAccCloudflarePageRule_ImportWithBrowserCacheTTL30 (14.50s)
=== RUN   TestAccCloudflarePageRule_ImportWithoutBrowserCacheTTL
--- PASS: TestAccCloudflarePageRule_ImportWithoutBrowserCacheTTL (13.57s)
=== RUN   TestAccCloudflarePageRule_Basic
--- PASS: TestAccCloudflarePageRule_Basic (11.76s)
=== RUN   TestAccCloudflarePageRule_FullySpecified
--- PASS: TestAccCloudflarePageRule_FullySpecified (10.29s)
=== RUN   TestAccCloudflarePageRule_ForwardingOnly
--- PASS: TestAccCloudflarePageRule_ForwardingOnly (9.50s)
=== RUN   TestAccCloudflarePageRule_ForwardingAndOthers
--- PASS: TestAccCloudflarePageRule_ForwardingAndOthers (3.97s)
=== RUN   TestAccCloudflarePageRule_DisableZaraz
--- PASS: TestAccCloudflarePageRule_DisableZaraz (9.79s)
=== RUN   TestAccCloudflarePageRule_Updated
--- PASS: TestAccCloudflarePageRule_Updated (18.90s)
=== RUN   TestAccCloudflarePageRule_CreateAfterManualDestroy
--- PASS: TestAccCloudflarePageRule_CreateAfterManualDestroy (17.44s)
=== RUN   TestAccCloudflarePageRule_UpdatingZoneForcesNewResource
--- PASS: TestAccCloudflarePageRule_UpdatingZoneForcesNewResource (18.32s)
=== RUN   TestAccCloudflarePageRule_CreatesBrowserCacheTTLIntegerValues
--- PASS: TestAccCloudflarePageRule_CreatesBrowserCacheTTLIntegerValues (10.90s)
=== RUN   TestAccCloudflarePageRule_CreatesBrowserCacheTTLThatRespectsExistingHeaders
--- PASS: TestAccCloudflarePageRule_CreatesBrowserCacheTTLThatRespectsExistingHeaders (14.65s)
=== RUN   TestAccCloudflarePageRule_UpdatesBrowserCacheTTLToSameValue
--- PASS: TestAccCloudflarePageRule_UpdatesBrowserCacheTTLToSameValue (18.41s)
=== RUN   TestAccCloudflarePageRule_UpdatesBrowserCacheTTLThatRespectsExistingHeaders
--- PASS: TestAccCloudflarePageRule_UpdatesBrowserCacheTTLThatRespectsExistingHeaders (17.65s)
=== RUN   TestAccCloudflarePageRule_DeletesBrowserCacheTTLThatRespectsExistingHeaders
--- PASS: TestAccCloudflarePageRule_DeletesBrowserCacheTTLThatRespectsExistingHeaders (17.09s)
=== CONT  TestAccCloudflarePageRule_Import
--- PASS: TestAccCloudflarePageRule_Import (14.10s)
PASS
ok  	github.com/cloudflare/terraform-provider-cloudflare/cloudflare	221.136s
?   	github.com/cloudflare/terraform-provider-cloudflare/tools/cmd/changelog-check	[no test files]
?   	github.com/cloudflare/terraform-provider-cloudflare/tools/cmd/maintainer-only-file-check	[no test files]
?   	github.com/cloudflare/terraform-provider-cloudflare/tools/cmd/tf-log-check	[no test files]
?   	github.com/cloudflare/terraform-provider-cloudflare/version	[no test files]
```

(note: I did redo all the page rules tests here to use unique identifiers to prevent accident collisions or test leaks).

**Comment by LANGERGabrielle** [comment]
Thank you for the quick merge !

**Pull request merged**